inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/convolution/convolution_kernel_bfyx_os_iyx_osv16.h

```diff
@@ -40,7 +40,12 @@ class ConvolutionKernel_bfyx_os_iyx_osv16 : public ConvolutionKernelBase {
             // Smaller # EU tends to be computation bounds.
             // In such case, using larger worksize will result in larger computational inefficiency
             // w.r.t the unalined output feature
-            return (params.output.Feature().v > 8 || !IsSIMDSizeSupported(params.engineInfo, 8)) ? 16 : 8;
+            if (params.output.Feature().v > 8 || !IsSIMDSizeSupported(params.engineInfo, 8)
```

Comment:
Android CI fails here now: https://openvino-ci.intel.com/blue/organizations/jenkins/private-ci%2Fie%2Fbuild-linux-android_x64/detail/build-linux-android_x64/65185/pipeline/
`error: '&&' within '||' [-Werror,-Wlogical-op-parentheses]
[2021-12-07T13:15:58.896Z]                || (params.output.GetDType() == Datatype::F16) && params.output.Feature().v == 8) {
[2021-12-07T13:15:58.896Z]                ~~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[2021-12-07T13:15:58.896Z] /home/jenkins/agent/workspace/private-ci/ie/build-linux-android_x64/b/repos/openvino/inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/convolution/convolution_kernel_bfyx_os_iyx_osv16.h:44:63: note: place parentheses around the '&&' expression to silence this warning
[2021-12-07T13:15:58.896Z]                || (params.output.GetDType() == Datatype::F16) && params.output.Feature().v == 8) {
[2021-12-07T13:15:58.896Z]                                                               ^`

Reply:
Fix: https://github.com/openvinotoolkit/openvino/pull/9067